src/js/components/organisms/hero-basic/hero-basic.models.ts

```diff
@@ -0,0 +1,3 @@
+/** @format */
+
+export interface HeroBasicProps {}
```

Comment:
If this file isn't used it should be removed.

Reply:
Whoops, forgot that it needs to be used.